pkg/services/ngalert/notifier/channels/kafka.go

```diff
@@ -125,36 +147,36 @@ func (kn *KafkaNotifier) SendResolved() bool {
 }
 
 func (kn *KafkaNotifier) buildBody(ctx context.Context, tmpl func(string) string, as ...*types.Alert) (string, error) {
-	bodyJSON := simplejson.New()
-	bodyJSON.Set("client", "Grafana")
-	bodyJSON.Set("description", tmpl(kn.settings.Description))
-	bodyJSON.Set("details", tmpl(kn.settings.Details))
+	record := kafkaRecord{}
```

Comment:
```suggestion
	var record kafkaRecord
```

Reply:
Adding `Client`, `Description` and `Details` in this same declaration could be an option too.

Reply:
the point was to change line-to-line